### PR TITLE
Migrate create react app to vite and change folder structure to clean up repo

### PR DESCRIPTION
**Pull request by Lord-Bubbles**
Make frontend and backend separate projects with own dependency to clean up repo

### CODE REVIEW

File: client/src/App.jsx
Comment (Lord-Bubbles) [inline client/src/App.jsx:1]: For now ignore these changes. These things will most likely be changed in future PRs

File: client/src/components/Header.jsx
Comment (Lord-Bubbles) [inline client/src/components/Header.jsx:1]: Ditto

File: client/src/components/MyMealPlan.jsx
Comment (Lord-Bubbles) [inline client/src/components/MyMealPlan.jsx:1]: Ditto

File: server/controllers/AuthController.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        },
+        message: 'New user created successfully!',
+      };
+    } else {
+      throw new Error('Error in user creation!');
+    }
+  }
+};
+
+export const fetchTokens = asyncHandler(async (req, res) => {
```
Comment (Lord-Bubbles) [inline server/controllers/AuthController.js:37]: Can ignore this function too

File: server/database/Calendar.js
Comment (Lord-Bubbles) [inline server/database/Calendar.js:1]: Ditto

File: server/controllers/AuthController.js
Comment (Lord-Bubbles) [inline server/controllers/AuthController.js:1]: ditto

**Review by chilimalone — APPROVED** [review]

**Review by drewhli — APPROVED** [review]
For future reference, if a file should be ignored for review, then that file shouldn't be included in the first place. If the file is needed to make things compile, or you want the placeholder for the future, then you can use an extremely basic template with simple declarations and TODOs.

**Pull request merged**